app/assets/stylesheets/base/main.scss

```diff
@@ -0,0 +1,39 @@
+@import '../config/import';
+
```

Comment:
This is a new file that will now contain some very fundamental basics for Crayons - mostly styling `body` element + some additions. It used to be part of scaffolds.scss but that file is really messy and very DEV oriented so I decided to take some crucial styling out of it.